WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -112,7 +112,7 @@ extension XCTestCase {
     }
 
     public func getCategory() -> String {
-        return "iOS Test"
+        return "Wedding"
```

Comment:
@rachelmcr – I've updated this to use a category from the test site – this is reflected in the updated mocks.